riscv-regs/Cargo.toml

```diff
@@ -7,5 +7,6 @@ edition = "2021"
 
 [dependencies]
 tock-registers = { version = "0.7" }
+riscv-decode = "0.2"
```

Comment:
Not related to this change, but typo in commit message.

Reply:
Done.